docs/salt.md

```diff
@@ -20,8 +20,16 @@ The salt-master is contacted by each salt-minion and depending upon the machine
 
 If you are running the Vagrant based environment, the **salt-api** service is running on the kubernetes-master.  It is configured to enable the vagrant user to introspect the salt cluster in order to find out about machines in the Vagrant environment via a REST API.
 
+## Standalone Salt Configuration on GCE
+
+On GCE, the master and nodes are all configured as [standalone minions](http://docs.saltstack.com/en/latest/topics/tutorials/standalone_minion.html). The configuration for each VM is derived from the VM's [instance metadata](https://cloud.google.com/compute/docs/metadata) and then stored in Salt grains (`/etc/salt/minion.d/grains.conf`) and pillars (`/srv/salt-overlay/pillar/cluster-params.sls`) that local Salt uses to enforce state.
+
+All remaining sections that refer to master/minion setups should be ignored for GCE. One fallout of the GCE setup is that the Salt mine doesn't exist - there is no sharing of configuration amongst nodes. This has ramifications later.
+
 ## Salt security
 
+(Not applicable on default GCE setup.)
```

Comment:
Fun optional nit: I think this would look great in italics. Feel free to disagree.